### ISSUE TITLE
What makes you think that enums are not global?

### ISSUE THREAD

**Issue by ParticlePeter** [ColonelThirtyTwo/dvulkan#5]
Check out DerelictGL3 or DerelictGLFW, global enums all over the place.  I strongly recommend to remove this behavior, this deviates unnecessarily from the original. You could alias VkResult as int, its nothing else in the vulkan.h header.
I read Manuzor's pull request, I disagree with his point, but think if it is an optional feature it would be ok. However, the bindings you ship with this repo should be as close as possible to the original.

In any case, thanks for the awesome work.

**Comment by ParticlePeter** [comment]
The readme.md misguided me and this issue, meanwhile I read your sources and found dub version directive DVulkanGlobalEnums. 
Firstly I recommend this behavior being the default and using a directive to specify non-global enums. Same reason as above, its closer to vulkan and the vulkan spec.
Secondly, this behavior should be properly explained in the readme.md

**Comment by ColonelThirtyTwo** [comment]
Apparently I didn't commit the readme changes that mention that version. Sorry about that.

Enums (actual enumerations, not the compile-time constant anonymous enumerations) in D store their values in their own namespace rather than globally. And since Vulkan used enums instead of `#define` macros, I decided to use them in the D bindings for the additional type safety. I didn't want to make the enumerations global, since in my opinion the namespacing of different enums leads to cleaner code, but after a request and me getting annoyed at typing out the enum name twice every time, I added the version.
